lightkurve/targetpixelfile.py

```diff
@@ -1012,23 +1014,82 @@ def from_fits_images(images, position=None, size=(10, 10), extension=None,
                                                n_rows=size[0],
                                                n_cols=size[1],
                                                target_id=target_id)
+
+        mid_img = images[int(len(images)/2)-1] # Find middle image to use as a reference
+        #Check if the image is a single FITS image, or a FITS file and get the appropriate PrimaryHDU or ImageHDU,
+        if isinstance(mid_img, fits.ImageHDU):
+            mid_hdu = mid_img
+        elif isinstance(mid_img, fits.HDUList):
+            mid_hdu = mid_img[extension]
+        else:
+            mid_hdu = fits.open(mid_img)[extension]
+        # Calculate reference WCS coords from the middle image. Get the middle pixel column and row.
+        wcs = WCS(mid_hdu)
+        pixelpos = skycoord_to_pixel(position, wcs=wcs)
+        column, row = int(np.round(pixelpos[0])), int(np.round(pixelpos[1]))
+
+        # Calculate the ra and dec of the center pixel of the middle image
+        skyposctr = pixel_to_skycoord(column, row, wcs=wcs)
+        ractr = skyposctr.ra.deg
+        decctr = skyposctr.dec.deg
+
+        factory.keywords = mid_hdu.header  # Get default keyword values from the first image
+
         for idx, img in tqdm(enumerate(images), total=len(images)):
+            #Check if an image is a single FITS image, or a FITS file, and return the appropriate PrimaryHDU or ImageHDU.
             if isinstance(img, fits.ImageHDU):
                 hdu = img
             elif isinstance(img, fits.HDUList):
                 hdu = img[extension]
             else:
                 hdu = fits.open(img)[extension]
-            if idx == 0:  # Get default keyword values from the first image
-                factory.keywords = hdu.header
+
+            # Get positional shift of the image compared to the reference file
+            wcs1 = WCS(hdu)
+            pixelpos1 = skycoord_to_pixel(position, wcs=wcs1)
+            poscorr1 = pixelpos1[0] - pixelpos[0]
+            poscorr2 = pixelpos1[1] - pixelpos[1]
+            # Add calculated pos corr shifts to the keywords dictionary
+            hdu.header['POSCORR1'] = pos_corr1
+            hdu.header['POSCORR2'] = pos_corr2
+
             if position is None:
                 cutout = hdu
             else:
                 cutout = Cutout2D(hdu.data, position, wcs=WCS(hdu.header),
                                   size=size, mode='partial')
-            factory.add_cadence(frameno=idx, flux=cutout.data, header=hdu.header)
```

Comment:
I am confused here as to why we need to get a wcs for the cutout. It looks to me like we aren't using it, as the header info passed to the factory is from the orginal file, not the new one. Thoughts?